### PR TITLE
Whipped into shape

### PR DESCRIPTION
**Pull request by zackkatz**

### CODE REVIEW

File: assets/src/css/blocks.scss [hunk 1]
```diff
@@ -0,0 +1,146 @@
+$color-1 : #e6f0f5;
+$color-2 : #ff0038;
+@import "../../../node_modules/react-datepicker/src/stylesheets/datepicker.scss";
+.react-datepicker-popper {
+	z-index : 5;
+}
+
+.gravity-view-shortcode-preview {
```
Comment (zackkatz) [inline assets/src/css/blocks.scss]: @mrcasual Please fix naming across the whole plugin to be `gravityview`, not `gravity-view`

File: blocks/setup.php [hunk 2]
```diff
@@ -1,113 +1,94 @@
...
+
+// Exit if accessed directly or Gutenberg is not enabled
+if ( ! defined( 'ABSPATH' ) || ! function_exists( 'register_block_type' ) ) {
+	exit;
+}
+
+/**
+ * Enqueue UI assets
+ */
+add_action( 'enqueue_block_editor_assets', function () {
```
Comment (zackkatz) [inline blocks/setup.php]: Please convert to not anonymous functions

File: blocks/setup.php [hunk 3]
```diff
@@ -1,113 +1,94 @@
...
+	wp_enqueue_style(
+		'gv-blocks-css',
+		GV_BLOCKS_PLUGIN_URL . $style,
+		array( 'wp-edit-blocks' ),
+		filemtime( GV_BLOCKS_PLUGIN_PATH . $style )
+	);
+
+	$views_list_array = array();
+
+	if ( class_exists( 'GVCommon' ) ) {
```
Comment (zackkatz) [inline blocks/setup.php]: Should none of the plugin should load if GravityView isn't loaded?

File: blocks/setup.php [hunk 4]
```diff
@@ -1,113 +1,94 @@
...
+	}
+
+	wp_localize_script(
+		'gv-blocks-js',
+		'GV_BLOCKS',
+		array(
+			'home_page' => home_url(),
+			'ajax_url'  => admin_url( 'admin-ajax.php' ),
+			'img_url'   => GV_BLOCKS_PLUGIN_URL . 'assets/img/',
+			'view_list' => $views_list_array
```
Comment (zackkatz) [inline blocks/setup.php]: End with comma

File: blocks/view-details/render.php [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<?php
+
+if ( ! function_exists( 'gv_blocks_render_view_details' ) ) {
```
Comment (zackkatz) [inline blocks/view-details/render.php]: This doesn't make any sense…the function is defined below. Do you think it was meant to say "If it _is_ defined, return"?

File: package.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "gravityview-blocks",
+  "version": "1.0.0",
+  "description": "Add GravityView layouts to Wordpress posts and pages using Gutenberg editor",
```
Comment (zackkatz) [inline package.json]: Word**P**ress

File: blocks/setup.php (hunk 3 above)
Comment (mrcasual) [reply blocks/setup.php]: @zackkatz are we going to merge this into core or keep as a separate "extension"?

**Pull request merged**